crate/build.rs

```diff
@@ -17,7 +17,7 @@ static NPM_CMD: &str = "npm";
 fn main() {
     println!("Build Trustify - build.rs!");
 
-    println!("cargo:rerun-if-changed={}", UI_DIR_SRC);
+    println!("cargo:rerun-if-changed={UI_DIR_SRC}");
```

Comment:
This change is not needed

Reply:
@carlosthe19916, if you want I could push that one in a separate PR as it's not related to this issue.
What do you think ?